src/app/child-dev-project/attendance/dashboard-widgets/attendance-week-dashboard/attendance-week-dashboard.component.html

```diff
@@ -12,36 +12,24 @@
   <mat-card-content>
     <table class="dashboard-table">
       <tr
-        *ngFor="let record of attendanceRecords; trackBy: recordTrackByFunction; let i=index;"
+        *ngFor="let rowGroup of dashboardRowGroups; trackBy: recordTrackByFunction; let i=index;"
         class="dashboard-table-row w-WARNING"
-        (click)="goToChild(record.child.getId())"
+        [routerLink]="['/child', rowGroup[0].childId]"
```

Comment:
We should soon figure out a general rule for the route names (#553). If somebody changes the name of the route of the child details view in the config, this code wont work any more

Reply:
:+1: